Tests/conf.json

```diff
@@ -1842,6 +1847,7 @@
         "McAfee NSM": "Nightly - Checking",
         "Forcepoint": "Nightly - Checking",
         "Phishme Intelligence": "Proxy has a non empty default value, will require fixing and merging",
+        "Cofense Intelligence": "Proxy has a non empty default value, will require fixing and merging",
```

Comment:
fix the problem instead of adding to unmockalbes.

Reply:
I think it works. Erased it from mock disable and it passed circle :)